app/decorators/step_decorator.rb

```diff
@@ -17,6 +17,10 @@ def action_name
     get_step_text(:execute_button)
   end
 
+  def waiting_text
```

Comment:
I'd expect to see a unit test for this / any method we create. 

If ProposalDecorator specs are hitting this, we should stub it out in those specs so they aren't dependent on the behavior in here.